web_tree_many2one_clickable/static/src/js/web_tree_many2one_clickable.js

```diff
@@ -47,7 +47,9 @@ openerp.web_tree_many2one_clickable = function(instance, local)
                     .filter([['key', '=', 'web_tree_many2one_clickable.default']])
                     .first()
                     .then(function(param) {
-                        self.use_many2one_clickable = (param.value == 'true');
+                        if (param) {
+                            self.use_many2one_clickable = (param.value == 'true');
+                        }
```

Comment:
Far from being an expert at Js, but could this work?:

    self.use_many2one_clickable = (param and param.value);